packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -151,11 +157,28 @@ class MediaQueryData {
   /// See also:
   ///
   ///  * [MediaQueryData], which provides some additional detail about this
-  ///    property and how it differs from [viewInsets].
+  ///    property and how it differs from [viewInsets] and [viewPadding].
   ///  * [SafeArea], a widget that consumes this padding with a [Padding] widget
   ///    and automatically removes it from the [MediaQuery] for its child.
   final EdgeInsets padding;
```

Comment:
On window, this is now a derived property.  I think we may want to make it so here as well.  /cc @HansMuller or @Hixie for input on that.

Reply:
Discussed this offline further - we have a direction now.